x-pack/solutions/security/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/insights/workflow_insights_scan.tsx

```diff
@@ -92,7 +96,17 @@ export const WorkflowInsightsScanSection = ({
         </EuiButton>
       </EuiFlexItem>
     );
+
+    if (!canWriteWorkflowInsights) {
+      return (
+        <EuiToolTip content={WORKFLOW_INSIGHTS.scan.noPermissions} position={'top'}>
```

Comment:
Are we displaying the "no permission" text tooltip with the Essentials tier in serverless? Wouldn't showing an "upselling" message make more sense in this scenario?

Reply:
The team has decided not to display upselling in either ESS or serverless environments for now.

Reply:
Okay, just be aware that users in (Serverless) Essential tier will see the "no privilege" tooltip here.

Reply:
This whole section won't be rendered because of the check here - https://github.com/elastic/kibana/pull/205088/files/7418c963025d20ecf205d4da1e002e863e929b3d#diff-31862d846041bb18fcd0f626034ccb58b6e5d91214f640f57940229728776551R198

Reply:
I see. We hide everything when a user does not have the "read" privilege. This is consistent in both scenarios (privileges, serverless tier). Looks good 👍